examples/texture_cubemap/source/texture_cubemap.cpp

```diff
@@ -0,0 +1,449 @@
+#include <gvk.hpp>
+#include <imgui.h>
+#include "image_resource.hpp"
```

Comment:
I can move the include to gvk.hpp, however there is a general issue with a convenience header like gvk.hpp. As it includes all other headers, any change to any header will cause a rebuild of all files that include gvk.hpp. This might be ok for small user projects, but for framework development and larger projects, this is very inefficient. This was also the reason why I included the header directly in the first place. 
I suggest to open a new issue that adresses this, namely by 
1) retaining gvk.hpp as a convenience header only for beginners and small user projects,
2) adapt the framework so that source files only include the header files they actually need to compile.
This should reduce the amount of recompilation following code changes to a minimum.

Reply:
I see. 

Indeed, the idea behind `gvk.hpp` is to make the framework usage 1) simple and 2) fast, where "fast" actually refers to compile times _for a framework user_. The whole `gvk.hpp` can be added as precompiled header (which is done by all of our example applications). That should make compilation very fast _if_ nothing in the framework changes.

Framework compile times can suffer from that structure, yes. I can see that. I appreciate your suggestions and maybe we should turn them into a new issue (May I ask you to create an issue for it?). If we restructure the framework in this manner, we should definitely also investigate C++20 modules!

Reply:
I have done some more research on precompiled headers in VC++ and opened a new issue (#95) with all the details. I have not looked into the details of C++20 modules yet, but have mentioned them briefly.

Reply:
Wow, that's a great issue + great description. Thank you!